backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/BaseGraphqlTest.java

```diff
@@ -212,35 +174,56 @@ protected ObjectNode runQuery(String queryFileName, ObjectNode variables, String
   /**
    * Run the query in the given resource file, check if the response has the expected error (either
    * none or a single specific error message), and return the {@code data} section of the response
-   * if the error was as expected. <b>NOTE</b>: Any headers that have been set on the {@link
-   * GraphQLTestTemplate} will be cleared at the beginning of this method: if you need to set them,
-   * modify the {{@link #setQueryHeaders(String)} method, or add another method that is called after
-   * it!
+   * if the error was as expected. <b>NOTE</b>: Any headers that have been set will be cleared at
+   * the beginning of this method
    *
    * @param queryFileName the resource file name of the query (to be found in
    *     src/test/resources/queries, unless a "/" is found in the filename)
    * @param operationName the operation name from the query file, in the event that the query file
    *     is a multi-operation document.
    * @return the "data" key from the server response.
-   * @throws AssertionFailedError if the response has errors
-   * @throws RuntimeException for unexpected errors
    */
   protected ObjectNode runQuery(
       String queryFileName, String operationName, ObjectNode variables, String expectedError) {
-    if (queryFileName != null && !queryFileName.contains("/")) {
-      queryFileName = "queries/" + queryFileName;
+    // change variables to map everywhere! seems like this will continue to give us issues
+    WebGraphQlTester webGraphQlTester =
+        this.graphQlTester
+            .mutate()
+            .headers(HttpHeaders::clear)
+            .headers(headers -> headers.setBearerAuth(getBearerAuth()))
+            .headers(httpHeaders -> httpHeaders.addAll(_customHeaders))
+            .build();
+
+    System.out.println(queryFileName);
+    GraphQlTester.Request<?> request = webGraphQlTester.documentName(queryFileName);
+
+    if (operationName != null) {
+      request.operationName(operationName);
     }
-    try {
-      setQueryHeaders();
-      GraphQLResponse response = _template.perform(queryFileName, operationName, variables);
-      assertEquals(HttpStatus.OK, response.getStatusCode(), "Servlet response should be OK");
-      _lastResponse = response.getRawResponse();
-      JsonNode responseBody = response.readTree();
-      assertGraphQLOutcome(responseBody, expectedError);
-      return (ObjectNode) responseBody.get("data");
-    } catch (IOException e) {
-      throw new RuntimeException(e);
+
+    if (variables != null) {
+      Gson gson = new Gson();
+      Type mapType = new TypeToken<Map<String, Object>>() {}.getType();
+      Map<String, Object> variableMap = gson.fromJson(variables.toString(), mapType);
```

Comment:
this reads all the variables from the `ObjectNode` and converts it to a `Map`, this was done so we dont change the `runQuery` signature and reduce the amount of changes in this PR. A follow up clean up PR will address changing the `runQuery` signature to accept a `Map` directly

Reply:
Can you add a ticket for that cleanup work Zedd? (Or maybe a central ticket tracking all the various followups we want to do)

Reply:
inc. the transfer to REST endpoints for multipart, for instance